Monika After Story/game/python-packages/battery/windows.py

```diff
@@ -0,0 +1,89 @@
+"""
+This module provides functions to check information for the battery and
+the AC Line on Windows systems.
+
+It may only work on Windows Server 2003 / Windows XP and above.
+
+For more information see the following:
+https://msdn.microsoft.com/en-us/library/windows/desktop/aa373232(v=vs.85).aspx
+"""
+
+import ctypes
+
+try:
+    from ctypes import wintypes
+except:  # pragma: no cover
+    pass
+
+
+def _system_power_status():
+    """
+    Return 'SYSTEM_POWER_STATUS' C structure with the values set by
+    GetSystemPowerStatus.
+    """
+    class SYSTEM_POWER_STATUS(ctypes.Structure):
+        """
+        This class is a representation of the SYSTEM_POWER_STATUS C structure
+        used by GetSystemPowerStatus.
+        """
+        _fields_ = [
+            ('ACLineStatus', wintypes.BYTE),
+            ('BatteryFlag', wintypes.BYTE),
+            ('BatteryLifePercent', wintypes.BYTE),
+            ('Reserved1', wintypes.BYTE),
+            ('BatteryLifeTime', wintypes.DWORD),
+            ('BatteryFullLifeTime', wintypes.DWORD),
+        ]
+
+    SPS_PTR = ctypes.POINTER(SYSTEM_POWER_STATUS)
+
+    GetSystemPowerStatus = ctypes.windll.kernel32.GetSystemPowerStatus
+    GetSystemPowerStatus.argtypes = [SPS_PTR]
+    GetSystemPowerStatus.restype = wintypes.BOOL
+
+    return SYSTEM_POWER_STATUS
```

Comment:
I actually forgot to run the "GetSystemPowerStatus" function which change the value of the SYSTEM_POWER_STATUS instance in memory.

```python
status = SYSTEM_POWER_STATUS()
GetSystemPowerStatus(ctypes.pointer(status))
return status
```